query/stdlib/influxdata/influxdb/rules_test.go

```diff
@@ -657,7 +491,6 @@ func TestPushDownFilterRule(t *testing.T) {
 	for _, tc := range tests {
 		tc := tc
 		t.Run(tc.Name, func(t *testing.T) {
-			t.Parallel()
```

Comment:
Making tests run serially isn't critical to tests passing right?

Reply:
I think what was happening was that we were re-using function expressions in different cases, and they were getting transformed in-place. Unless we wrap those exprs in closures, the test will have issues.

Reply:
I guess what I'm saying is that I'm okay with not letting them run in parallel, and leaving the test as-is.

Reply:
That works for me.